Resources/Computing/LocalComputingElement.py

```diff
@@ -215,20 +220,29 @@ def getCEStatus(self):
   def getJobStatus(self, jobIDList):
     """ Get the status information for the given list of jobs
     """
-    stampList = []
+    resultDict = {}
+    jobDict = {}
+
+    # Extract the batch job ID from the full DIRAC job ID
     for job in jobIDList:
       stamp = os.path.basename(urlparse(job).path)
-      stampList.append(stamp)
+      jobDict[stamp] = job
+    stampList = jobDict.keys()
```

Comment:
Please don't use `.keys()` (python 3...). You can use 
```suggestion
    stampList = list(jobDict)
```